app/assets/javascripts/nested_related_items.js

```diff
@@ -13,12 +13,21 @@
         $(this).removeClass('open');
         _this.addToggleLink($(this));
       });
+
+      _this.addToggleAll();
+      _this.toggleAll();
     },
 
     listItems: function(list) {
       return list.find(this.options.itemSelector);
     },
 
+    addToggleAll: function() {
+      var title = $('dt[title="Contains"]');
+      $(title).append('<a href="javascript:;" id="toggleAll" class="mods_display_related_item_label">Expand all</a>');
+      $(title).attr('id', 'mods_display_related_item_dt');
```

Comment:
Can we just call `.append` and `.attr` directly on `title` instead of wrapping it in `$()`?

Reply:
👍